src/main.rs

```diff
@@ -300,29 +263,6 @@ async fn main() -> eyre::Result<()> {
     Ok(())
 }
 
-fn init_tracing(endpoint: &str) {
-    global::set_text_map_propagator(TraceContextPropagator::new());
-    let provider = opentelemetry_otlp::new_pipeline()
-        .tracing()
-        .with_exporter(
-            opentelemetry_otlp::new_exporter()
-                .tonic()
-                .with_endpoint(endpoint),
-        )
-        .with_trace_config(Config::default().with_resource(Resource::new(vec![
-            opentelemetry::KeyValue::new("service.name", "rollup-boost"),
-        ])))
-        .install_batch(opentelemetry_sdk::runtime::Tokio);
-    match provider {
-        Ok(provider) => {
-            let _ = global::set_tracer_provider(provider);
-        }
-        Err(e) => {
-            error!(message = "failed to initiate tracing provider", "error" = %e);
-        }
-    }
-}
-
 async fn init_metrics_server(addr: SocketAddr, handle: PrometheusHandle) -> eyre::Result<()> {
```

Comment:
can move this function to the metrics file?

Reply:
This is done in one of my other PRs already :)